docs/api/introduction/api-endpoints.md

```diff
@@ -0,0 +1,34 @@
+---
+id: api-endpoints
+title: API Endpoints
+keywords:
+  - api
+  - introduction
+  - endpoints
+description: Zilliqa API Endpoints
+---
+
+---
+
+You can use the following endpoints to interact with the Zilliqa API, either via EVM or the
+[Zilliqa JSON API](./api-introduction). Note that the EVM endpoint may change in the future.
```

Comment:
I'd say lets just remove this line and save the reader some anxiety. Realistically once we've started serving EVM endpoints at this address, it will be hard to stop without significant pain and the reward is minimal.
```suggestion
[Zilliqa JSON API](./api-introduction).
```